### ISSUE TITLE
How to add more users

### ISSUE THREAD

**Issue by nicoesat** [bioinformatics-ua/dicoogle-learning-pack#11]
How do I create new users for Dicoogle?
There doesn't seem to be an UI to do so.  I tried editing the users.xml file, but I don't know what type of hash is used to store the password.

According to the logs there should also be a roles.xml file.  What should it look like? What can I do with it?

Nico

**Comment by nicoesat** [comment]
Wow, that was fast!

Thanks, that is exactly what I needed!

Nico

**Comment by nicoesat** [comment]
But unfortunately the changes are not saved in the users.xml file so they don't survive a reboot.

**Comment by Enet4** [comment]
Well, that is odd. The changes are expected to be saved immediately after issuing a new user, so this might be a bug. One that seems to have been fixed in the latest development versions (v3), but the changes made here are quite significant. This might indeed be related with issue bioinformatics-ua/dicoogle#361, since we've spotted similar routines for reading and writing XML configuration files.

Considering that we would like to make migrations to v3 as smooth as possible in the future, it might be worth fixing this bug in v2 as well.
